### ISSUE TITLE
Latest 1.8 built 15210-v1.5-4248-noads.apk fails to update

### ISSUE THREAD

**Issue by thespianuk** [Trixarian/NetherSX2-patch#46]
Platform: Ayaneo Pocket Air Mediatek D1200 Android 12
Previously installed version: 1.7 Builder generated apk.

Downloaded builder v1.8 on Windows 11.  Ran batch file.  Patched version created.  Moved to SD card of Pocket Air. Moved to internal storage.  Tap file, asks if I want to update.  Tap yes.  Grey notification appears:

Something went wrong.  App not installed.

sem

**Comment by ThunderDragonRandy** [comment]
Yeah, it wouldn't update for me unless I uninstalled AetherSX2/NetherSX2 first.

1. [Optional but recommended] First of all, backup files important to you to another folder just in case as you'll have to uninstall the version you currently have installed to install the newest version.
2. Before installing, once you backed up your files elsewhere, clear all cache and data in the app's settings in Settings.
3. When uninstalling the previous version, DO NOT checkmark "Keep All (Remaining) Data" when installing the newest version as the installer will think you still have AetherSX2/NetherSX2 installed and the newest version will NOT install.
4. Install the newest version. You should have no problems installing the newest version now, but...
5. [Only if you screw up] IF by chance you accidentally select to "keep data" when uninstalling the previous version of AetherSX2/NetherSX2 (like I did when I was trying to update too), hope you have a computer and a cable to connect your PC too, and follow this [Reddit post's instructions](https://www.reddit.com/r/revancedapp/comments/xvx3g8/comment/j4lz2oi/?utm_source=reddit&utm_medium=web2x&context=3) by [amarhany20](https://www.reddit.com/user/amarhany20/) to remove remaining data that thinks the app is still installed as this is how I fixed that issue.
